app/views/patients/_abortion_information.html.erb

```diff
@@ -35,6 +37,16 @@
         <% end %>
       </div>
     </div>
+
+    <div class="row">
+      <div class="col-sm-12" id="clinic_finder_container">
+
+        <%= link_to display_clinic_finder_path, class: 'display_button', remote: true do %>
+        <h3>Find clinics</h3>
```

Comment:
The pop open style from a call works well here, but if it winds up being easier we should feel free to just statically include it on page load and trigger it with a `hide`, and just render the `results` part. Might be easier to hook jquery into that way. But I'm definitely good either way as long as it works.